frontend/public/components/_environment.scss

```diff
@@ -2,3 +2,11 @@
   border-top: 1px solid #ddd;
   padding-top: 20px;
 }
+
+.environment-alert {
+  padding-left: 15px;
```

Comment:
I don't think this should be necessary.... Can you try taking the `row` style off the div and see if it's still needed?

Reply:
Yeah this is needed or the dropdown gets right aligned after the alert on the pods page.

Reply:
OK is there some flexbox stuff going on here?

This seems like a sign that we're not using the bootstrap grid styles correctly... Maybe @rhamilto or @sg00dwin can help.

Reply:
Its ok, I similar classes to align the component as other components on the page without the need for the padding. Adding it to the fixes on the next commit.